server/index.ts

```diff
@@ -5,9 +5,27 @@ import barChart from "./data/barChart.ts";
 import pieChart from "./data/pieChart.ts";
 import cors from "cors";
 import dotenv from "dotenv";
+import DescopeClient from "@descope/node-sdk";
+import { parseCookies } from "./authHelpers.ts";
+import { RequestContext } from "./types.ts";
 
 dotenv.config();
 
+declare global {
+  namespace Express {
+    interface Request {
+      context: RequestContext;
+    }
+  }
+}
+
+// Make sure you add your project ID in the .env file
+
+// TODO: Implement this
+const clientAuth = DescopeClient({
+  projectId: "P2OEsPZdWHN2CkaERPhpTd8M25aR",
```

Comment:
Remove project IDs

Reply:
Maybe I'll put a .env so people can change it from the .env instead of in the code. Otherwise, the app may crash when they try to run it.